aea/cli/eject.py

```diff
@@ -54,6 +56,10 @@
 from aea.helpers.base import find_topological_order, reachable_nodes
 
 
+IMPORT_TEMPLATE_1 = "from packages.{author}.{type}.{name}"
+IMPORT_TEMPLATE_2 = "import packages.{author}.{type}.{name}"
```

Comment:
Move these to utils constants or configurations constants (latter might be better as it can be used outside cli).